### PR TITLE
Clean up unnecessary code and assets

### PR DESCRIPTION
**Pull request by scheibo**

### CODE REVIEW

File: web-app/package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-   "name": "GoatControllerFrontEnd",
-   "version": "1.9.0",
-   "description": "The next great yugioh sim",
    "private": true,
-   "main": "src/index.js",
    "dependencies": {
+      "@babel/cli": "7.10.1",
```
Comment (acpennington) [inline web-app/package.json:4]: Why do we need babel in production? I was under the impression that this was always a dev dependency.

Comment (scheibo) [reply web-app/package.json:4]: If anything, babel is prod only? Babel is what does the translation to legacy JS/adding of polyfills etc which is mostly irrelevant in development where we can always just run the latest and greatest browser.

Arguably, the `@babel/cli` package specifically can be dev only, but separating out prod/dev/optional node dependencies is kind of pedantic in a private node package (its definitely relevant if youre publishing the package, but has negligible value in this sort of situation. are you a) actually using `npm install --production` religiously in prod and b) care particularly about the X KB of `@babel/cli` being installed on the prod server?)

Comment (acpennington) [reply web-app/package.json:4]: I'm no babel expert. Remember, there is no real "server" here since we're doing JAMstack/serverless. The way that deployment works is npm run build -> ship the build folder to S3. Since "npm run build" is technically only done in dev, I don't see how we'll need babel in prod. It's not as though any of our frontend has "import babel" in it.

The more prod dependencies we have, doesn't that mean the larger our final build size will be? Or do I just have no idea how any of this works?

Comment (scheibo) [reply web-app/package.json:4]: Oh - I didn't realize you were just uploaded the built folder. Then yes, you don't need `babel` in prod, but because of the way you deploy, you actually don't need *anything* in prod. *ALL* dependencies are 'dev' dependencies here - the dependencies are only required to produce `build` (which you do in development), and after you have built webpack has already done treeshaking and bundling to ensure only things actually used when serving the app will get used.

tl;dr: it literally does not matter at all whether we call all the dependencies dev dependencies or all the dependencies prod depenedencies, webpack does not care, and webpacks output is all that matters in prod.
